### PR TITLE
1800/feat/dropdown-occupation

### PR DESCRIPTION
**Pull request by barnahum8**
Change occupation radio group, to drop down list.

### CODE REVIEW

**Review by Noam6421 — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx [hunk 1]
```diff
@@ -174,42 +173,31 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
...
-                                )
+                                return (<FormControl variant='outlined' className={classes.occupationSelect}>
+                                    <Select
+                                        {...props}
+                                        placeholder={OCCUPATION_LABEL}
+                                        onChange={(event) => {
+                                            props.onChange(event.target.value)
+                                        }}>
+                                        {
+                                            occupations.map((occupation) => (
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:191]: Is occupation guaranteed to be empty? What happenes if the get /occupation API call fails?

**Review by Noam6421 — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts [hunk 2]
```diff
@@ -5,6 +5,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     form: {
         padding: '1vh 1.3vw 2.5vh 1.3vw',
     },
+    occupationSelect: {
+        width: '14vw',
+        height: '2vw'
+    },
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts:11]: Why both vw and a grid wrapper? If the item is gridded give it fullWidth and if it's controlled by vw, don't grid it

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx [hunk 3]
```diff
@@ -174,42 +173,31 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
                 </Grid>
                 <Grid item>
                     <Grid container justify='space-between'>
-                        <FieldName xs={7} fieldName={RELEVANT_OCCUPATION_LABEL}/>
-                        <Grid item xs={5}>
+                        <FieldName xs={5} fieldName={OCCUPATION_LABEL}/>
```
Comment (chenFriedman) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:176]: the sizes switched in purpose?

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts [hunk 4]
```diff
@@ -5,6 +5,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     form: {
         padding: '1vh 1.3vw 2.5vh 1.3vw',
     },
+    occupationSelect: {
```
Comment (chenFriedman) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts:8]: maybe you can show me and we can get this css by the define the grid size

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx (hunk 3 above)
Comment (barnahum8) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:176]: yes, the drop down is different size from radio button, and also the label has become shorter

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx (hunk 1 above)
Comment (barnahum8) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:191]: basically, in the radio group they also didn't check the occupations, so I kept it the same.

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts (hunk 4 above)
Comment (barnahum8) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts:8]: 👍

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:191]: 🤷

**Review by chenFriedman — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts (hunk 2 above)
Comment (barnahum8) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx (hunk 1 above)
Comment (barnahum8) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx]: fixed anyway

**Review by Noam6421 — APPROVED** [review]

**Review by y-golde — APPROVED** [review]

**Pull request merged**